app/templates/admin/createEvent.html

```diff
@@ -26,62 +26,65 @@
 {% set eventPast = event.isPastStart if not isNewEvent else False %}
 {% set tabName = 'edit' %}
 
-
 {% block scripts %}
-  {{super()}}
+  {{super()}} 
+  <!-- Include jQuery -->
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <!-- Include jQuery UI (required for datepicker) -->
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
This shouldn't work here, but maybe the browser is letting us get away with it. It should go in the styles block if necessary.

Reply:
moved into style block